src/msi/msi_token.go

```diff
@@ -208,3 +224,31 @@ func GetTokenFromCache(resource string, msiIdentity string) string {
 	}
 	return ""
 }
+
+func RefreshTokens() {
+	log.Info("Refreshing MSI Tokens, number of identities: " + strconv.Itoa(len(tokenStore)))
+	tokenMutex.RLock()
+	identities := make(map[string][]string, len(tokenStore))
+	for identity, tokenMap := range tokenStore {
+		identities[identity] = make([]string, len(tokenMap))
+		count := 0
```

Comment:
since it is a slice, append could be used, increasing the readability.

Reply:
This has been done this way all over the code, didn't want to digress.